flow-typed/npm/react-native-onesignal_vx.x.x.js

```diff
@@ -14,134 +14,5 @@
  */
 
 declare module 'react-native-onesignal' {
-  declare export type OneSignalOpenResult = {
```

Comment:
… I should try undoing this removal

Reply:
Looks like it was removed in the big edf29dec1a6ec3c6b609cc9958fabe05bd03b5a2 (flow-typed update) commit. I should be able to put it back with no problems.

Reply:
Resolved locally. Will push once the queue goes down.